RT-9: Disable IPv6 ND Router Advertisement

priyaranjannanda opened the pull request:
This is initial commit forRT-9: Disable IPv6 ND Router Advertisement. 
Resolves https://github.com/openconfig/featureprofiles/issues/1844 .

sachendras commented:
Since there is a conflict, please pick RT-9 for this test and include https://github.com/openconfig/featureprofiles/blob/main/testregistry.textproto in your change.

priyaranjannanda commented:
> Since there is a conflict, please pick RT-9 for this test and include https://github.com/openconfig/featureprofiles/blob/main/testregistry.textproto in your change.

Thanks Sachendras. I have updated the test case number in README and metadata.textproto. Updated the change in  text registry.textproto.
https://github.com/openconfig/featureprofiles/pull/2492

mohanasm commented:
/gcbrun

sezhang2 commented on feature/interface/ip/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/disable_ipv6_nd_ra_test.go:
```diff
@@ -0,0 +1,236 @@
+// Copyright 2022 Google LLC
```
change to 2024

priyaranjannanda replied:
Fixed it in the latest commit.

sezhang2 commented on feature/interface/ip/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/disable_ipv6_nd_ra_test.go:
```diff
@@ -0,0 +1,236 @@
+	return i
+}
+
+// configureOTG configures OTG interfaces to send and recieve ipv6 paxckets
```
Fix the typo paxckets

priyaranjannanda replied:
Fixed it in the latest commit.

sezhang2 commented:
Please also resolve conflicts

sezhang2 reviewed (dismissed):
A new deviation ipv6_router_advertisement_config_unsupported is added and tracked by b/316173974.

Please address some minor comments and resolve conflicts.

arulkumarsekar reviewed (approved):
re approving
